src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/zulaman/zulamanScripts.cpp

```diff
@@ -1174,16 +1174,16 @@ struct npc_tanzarAI : public ScriptedAI
 
                 switch (m_uiHelpShoutCounter)
                 {
-                    case 0:
+                    case 1:
```

Comment:
Nice try, but those are possible values of `m_uiHelpShoutCounter`, point indexes are a bit further down the code. 😉

Reply:
Should be fixed with the latest commit